### PR TITLE
Reviews1

### PR DESCRIPTION
**Pull request by heltan**
add stars and thumbnail images for reviews

### CODE REVIEW

File: coverage/lcov-report/client/src/components/ratingsReviews/index.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
+
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>Code coverage report for client/src/components/ratingsReviews</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="../../../../prettify.css" />
+    <link rel="stylesheet" href="../../../../base.css" />
```
Comment (Louis-La) [inline coverage/lcov-report/client/src/components/ratingsReviews/index.html:9]: Good separation of CSS files

File: client/src/components/ratingsReviews/ratings.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import axios from 'axios';
+import moment from 'moment';
  import Images from './images.jsx';
  import Stars from './stars.jsx';
```
Comment (Louis-La) [inline client/src/components/ratingsReviews/ratings.jsx:5]: Some spaces to fix for next time

File: client/src/components/ratingsReviews/stars.jsx [hunk 3]
```diff
@@ -13,7 +13,36 @@ class Stars extends React.Component {
...
+        let emptyStars = Math.floor(5 - fullStars - quarterStar);
+        if (emptyStars > 0) {
+            for (let i = 0; i < emptyStars; i++) {
+                fullStarDiv.push(<span className= "fa fa-star full-star-empty"></span>)
+            }
+        }
+        let starDiv = <div key={this.props.starKey + 'star'}>{fullStarDiv}</div>
+        this.setState({star: starDiv})
+    }
+    getQuarterStarAmount(value) {
```
Comment (isaack87) [inline client/src/components/ratingsReviews/stars.jsx:38]: maybe make this a switch case or else chain

**Review by Louis-La — COMMENTED** [review]
Code looks good. Very clean. Some spacings and indents can be fixed for next time.

File: client/src/components/ratingsReviews/stars.jsx [hunk 4]
```diff
@@ -13,7 +13,36 @@ class Stars extends React.Component {
         let rating = this.props.rating;
         //get how many full stars there should be by rounding down the nearest num
         let fullStars = Math.floor(rating);
+        //get the quarterStarAmount
+        let quarterStar = this.getQuarterStarAmount(rating - fullStars);
+        //loop through full stars to create full star div
+        let fullStarDiv = []
+        for (let i = 0; i < fullStars; i++) {
```
Comment (isaack87) [inline client/src/components/ratingsReviews/stars.jsx:20]: Use forEach

**Review by isaack87 — APPROVED** [review]

**Pull request merged**